### DISCUSSION TITLE
Welcome to BusinessCentral.LinterCop Discussions!

### DISCUSSION THREAD

**Discussion by StefanMaron** [StefanMaron/BusinessCentral.LinterCop#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fvet** [comment]
Hi @StefanMaron ,

I like the initiative. Where do you draw the line between earlier / similar suggested ideas to MS for their standard AL CodeCop and the LinterCop ideas?

**Reply by StefanMaron** [reply]
I think this needs to be a case to case decision. If a rule makes absolute sense, why not implementing it until MS covers this? 😎
